src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,21 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
```

Comment:
> On a related note, as long as we are creating queues just for demo course creation, why not put the entire demo course creation in the queue?

I'll answer this. @ablyx actually had that idea before as well. One of our tests rely on the end result of the demo course creation, in particular when the end of the operation is signified by the page showing the table entry:
![screen shot 2017-02-21 at 2 35 00 am](https://cloud.githubusercontent.com/assets/7261051/23137699/6c2714d0-f7de-11e6-9e85-df5051080de9.png)
If we move everything to task queue, it solves the entire 1-minute limit issue, but we lose testability because there is no telling when the operation will finish, and subsequent tests _need_ the created entities (demo course, demo instructors, demo students, demo ...) to be put in place.